pyarlo/camera.py

```diff
@@ -69,7 +74,6 @@ def unseen_videos(self):
         self.update()
         return self._attrs.get('mediaObjectCount')
 
-    @property
     def unseen_videos_reset(self):
```

Comment:
**BREAKING:** this felt like more of an action than a property, so i made it a method.

Reply:
+1 totally agreed!  Would be nice to cover this on a unittest.